lib/ReactViewModels/ViewState.ts

```diff
@@ -418,13 +418,15 @@ export default class ViewState {
       (disclaimerVisible) => {
         this.isMapFullScreen =
           disclaimerVisible ||
+          terria.elements.get("show-workbench")?.visible === false ||
           terria.userProperties.get("hideWorkbench") === "1" ||
           terria.userProperties.get("hideExplorerPanel") === "1";
       }
     );
 
     this._isMapFullScreenSubscription = reaction(
       () =>
+        terria.elements.get("show-workbench")?.visible === false ||
         terria.userProperties.get("hideWorkbench") === "1" ||
         terria.userProperties.get("hideExplorerPanel") === "1",
```

Comment:
The interaction between `show-workbench` (from elements), and `hideWorkbench` (from hash user properties) is a bit confusing. Ideally this is unified at some point, but I think for the moment we just need to make the behaviours consistent